agent/fw_drupal8.c

```diff
@@ -752,6 +797,66 @@ void nr_drupal_version() {
 }
 
 void nr_drupal8_enable(TSRMLS_D) {
+  int retval;
+  zend_class_entry* drupal_ce = NULL;
+  zend_class_entry* symfony_ce = NULL;
+  bool inject = true;
+
+  if (NRINI(drupal_page_cache_naming)) {
+    drupal_ce = nr_php_find_class("Drupal\\Core\\Routing\\RouteMatch");
+
+    if (NULL == drupal_ce) {
+      inject = false;
+      nrl_warning(NRL_FRAMEWORK, "Missing Drupal RouteMatch Class");
+    }
+
+    if (inject) {
```

Comment:
Can this be an `else` for the conditional on line 808?

Reply:
aebfeebf39718d891d7e03abb7e8672c4ca05efd